ownership-chain/precompile/evolution-collection-factory/src/tests.rs

```diff
@@ -144,3 +144,23 @@ fn create_collection_inserts_bytecode_to_address() {
 		assert!(AccountCodes::<Test>::get(&collection_address) == REVERT_BYTECODE);
 	});
 }
+
+#[test]
+fn test_expected_cost_create_collection() {
+	new_test_ext().execute_with(|| {
+		let input = EvmDataWriter::new_with_selector(Action::CreateCollection)
+			.write(Address(H160([1u8; 20])))
+			.build();
+
+		// Expected weight of the precompile call implementation.
+		// Since benchmarking precompiles is not supported yet, we are benchmarking
+		// functions that precompile calls internally.
+		//
+		// The weight of this precompile call is calculated as:
+		// `create_collection` weight + insert account bytecode for the collection + log costs
```

Comment:
log costs are also measured ? I cannot see where this metering is done in the `lib.rs` file